src/tests/crypto.edn

```diff
@@ -0,0 +1,135 @@
+(def _args (eval `(assoc {} ~@*command-line-args*)))
+(def test-rpcs (if (contains? _args "--rpcs") (read-string (get _args "--rpcs")) false))
+(def rpc-server "http://172.29.35.84:9933")
+; node: cargo run -- --dev --tmp --rpc-external --rpc-port 9933 --rpc-cors all --ws-external --enable-offchain-indexing 1 --rpc-methods=Unsafe
+
+(defnode root)
+(defchain main
+  ; test from hex string
+  "0x9d61b19deffd5a60ba844af492ec2cc44449c5697b326919703bac031cae7f60"
+  (| (Sr25519.PublicKey) (ToHex)
+     (Assert.Is "0x44a996beb1eef7bdcab976ab6d2ca26104834164ecf28fb375600576fcc6eb0f" true))
+  (| (Ed25519.PublicKey) (ToHex)
+     (Assert.Is "0xd75a980182b10ab7d54bfed3c964073a0ee172f3daa62325af021a68f707511a" true))
+  (| (ECDSA.PublicKey) (ToHex)
+     (Assert.Is "0x048db55b05db86c0b1786ca49f095d76344c9e6056b2f02701a7e7f3c20aabfd913ebbe148dd17c56551a52952371071a6c604b3f3abe8f2c8fa742158ea6dd7d4"))
+  ; test from bytes
+  (HexToBytes)
+  (| (Sr25519.PublicKey) (ToHex)
+     (Assert.Is "0x44a996beb1eef7bdcab976ab6d2ca26104834164ecf28fb375600576fcc6eb0f" true))
+  (| (Ed25519.PublicKey) (ToHex)
+     (Assert.Is "0xd75a980182b10ab7d54bfed3c964073a0ee172f3daa62325af021a68f707511a" true))
+  (| (ECDSA.PublicKey) (ToHex)
+     (Assert.Is "0x048db55b05db86c0b1786ca49f095d76344c9e6056b2f02701a7e7f3c20aabfd913ebbe148dd17c56551a52952371071a6c604b3f3abe8f2c8fa742158ea6dd7d4"))
+
+  "caution juice atom organ advance problem want pledge someone senior holiday very"
```

Comment:
Is this your secret passphrase 😄?

Reply:
haha,
nah copied from some substrate web page ;)